crrExistingObjects.js

```diff
@@ -272,9 +322,7 @@ function triggerCRROnBucket(bucketName, cb) {
                     log.error('error listing object versions', { error: err });
                     return done(err);
                 }
-                const versions = data.DeleteMarkers
-                    ? data.Versions.concat(data.DeleteMarkers) : data.Versions;
-                return _markPending(bucket, versions, err => {
+                return _markPending(bucket, data.Versions.concat(data.DeleteMarkers), err => {
```

Comment:
we should not concat if `data.DeleteMarkers` is null?
e.g. keep the older code or `data.Versions.concat(data.DeleteMarkers || [])` ?